data/hai/hai reveal 4 middle.txt

```diff
@@ -1498,7 +1498,56 @@ mission "Hai Reveal [A11-B] Permanent Fleets"
+mission "Hai Reveal [A11-C] Outgoing Election Update 1"
+	landing
+	source
+		government "Hai"
+		not planet "Hai-home"
+		attributes "spaceport"
+	to offer
+		has "Hai Reveal [A11] Sound the Alarm: offered"
+	on offer
+		conversation
+			`The spaceport appears to be operating on a skeleton staff with an increased security presence. Plenty of ships are still coming and going, but they seem to be here strictly on business. The usual milling of tourists, haggling of merchants, and perusal of jobs all appear to have been suspended in favor of efficient boarding and disembarking of Hai with places to go and goods with places to be. The displays in the main thoroughfare have been switched over to wall-to-wall election coverage and the rest of the non-essential sections of the spaceport have been closed.`
+			`	Some of the restaurants that are currently closed have set up temporary stands near the displays with the most standing room in the vicinity, and are serving food to those who have no choice but to be here for some duration of time.`
+			choice
+				`	(Get some food.)`
+				`	(Ask around for an update.)`
+					goto ask
+				`	(Return to your ship.)`
+					goto leave
+			`	The stalls have a range of tasty Hai and Hai-human fusion cuisines in disposable takeaway trays. At first glance they appear to be some sort of cardboard equivalent, but a Hai near you finishes their meal and then begins nibbling on the tray afterwards. It would seem that they are edible.`
+			`	The election coverage is in Hai, naturally, but there's a strong sense that people are still scrambling to have relevant commentary on such short notice and with no prior opinion polling in lead-up. It certainly seems like things are heated. Different charts being presented for the same areas seem to suggest that there's a great deal of controversy and some very strong opinions being thrown around. A number of the Hai nearby make some sounds of concern occasionally, but they do not stay around to comment. Unfortunately, nothing comes up in your language by the time you've finished your food.`
+			choice
+				`	(Eat the tray.)`
+					goto tray
+				`	(Return to your ship.)`
+					goto leave
+			label tray
+			`	The tray is hard and difficult to break. It does have a remarkably intriguing combination nutty and herby flavor that would probably be quite enjoyable, but you don't have the teeth for it.`
+			choice
+				`	(Ask around for an update.)`
+					goto ask
+				`	(Return to your ship.)`
+					goto leave
+			label ask
+			`	When you ask around, a lot of Hai don't seem to be very interested in talking to you. One in particular looks like they recognized you and actively avoided you. After you've approached a half dozen Hai unsuccessfully, though, one takes pity and comes over towards you. He appears to be of middling years and extends a paw to shake your hand in a friendly manner.`
```

Comment:
```suggestion
			`	When you ask around, a lot of Hai doesn't seem to be very interested in talking to you. One, in particular, looks like they recognized you and actively avoided you. After approaching a half dozen Hai unsuccessfully, one takes pity and comes to you. He appears to be of middling years and extends a paw to shake your hand in a friendly manner.`
```

Reply:
"A lot of Hai do not seem" is correct, "a lot of Hai does not seem" isn't.

Reply:
With Bene's fix
```suggestion
			`	When you ask around, a lot of Hai don't seem to be very interested in talking to you. One, in particular, looks like they recognized you and actively avoided you. After approaching a half dozen Hai unsuccessfully, one takes pity and comes to you. He appears to be of middling years and extends a paw to shake your hand in a friendly manner.`
```